tests/specs/ins/harvesting.e2e.js

```diff
@@ -110,6 +110,53 @@ describe('ins harvesting', () => {
     })
   })
 
+  it('should submit Marks', async () => {
+    const testUrl = await browser.testHandle.assetURL('marks-and-measures.html', { init: { performance: { capture_marks: true, capture_measures: false } } })
+    await browser.url(testUrl).then(() => browser.waitForAgentLoad())
+
+    const [[{ request: { body: { ins: insHarvest } } }]] = await Promise.all([
+      insightsCapture.waitForResult({ totalCount: 1 })
+    ])
+
+    expect(insHarvest.length).toEqual(2) // this page sets two marks
+    expect(insHarvest[0]).toMatchObject({
+      entryDuration: 0,
+      eventType: 'BrowserPerformance',
+      entryName: 'before-agent',
+      pageUrl: expect.any(String),
+      timestamp: expect.any(Number),
+      entryType: 'mark'
+    })
+    expect(insHarvest[1]).toMatchObject({
+      entryDuration: 0,
+      eventType: 'BrowserPerformance',
+      entryName: 'after-agent',
+      pageUrl: expect.any(String),
```

Comment:
Minor observation/Qs:

- Would we also want to test for `currentUrl`?  (i see it in the payload)
- This q is mostly out-of-scope but I started wondering about it while looking at the payloads. It seems like `currentUrl` and `pageUrl` are repetitive amongst the UserActions, as well as performance measures.  Would a single payload contain data from other urls / would we ever consider factoring out common url values within a payload to shrink down payload size?

Reply:
Discussed this on Slack yesterday - `currentUrl` may differ due to soft navs but `pageUrl` should theoretically stay the same.  So maybe it's something to look into in the future, should the need arise.

`currentUrl` is technically in the payload, but i don't consider it blocking or anything.  So marking as resolved.